graphql/src/main/java/io/micronaut/configuration/graphql/DefaultGraphQLInvocation.java

```diff
@@ -35,9 +36,9 @@
  * @author Marcel Overdijk
  * @author Graeme Rocher
  * @author James Kleeh
- * @since 1.0
- * @see GraphQLExecutionInputCustomizer#customize(ExecutionInput, HttpRequest)
+ * @see GraphQLExecutionInputCustomizer#customize(ExecutionInput, HttpRequest, MutableHttpResponse)
  * @see GraphQL#executeAsync(ExecutionInput.Builder)
+ * @since 1.0
```

Comment:
Why is this moved below @see ?

Reply:
Sorry, it was IntelliJ IDEA auto-format.